cls/SourceControl/Git/Extension.cls

```diff
@@ -49,7 +49,11 @@ Method UserAction(Type As %Integer, Name As %String, InternalName As %String, Se
     if Type = 1, Name = 5 {
         // reroute to Status menu option
         set Name = "%SourceMenu,Status"
-    } 
+        if (settings.gitUserName = "") || (settings.gitUserEmail = "") {
+            set Target = "Warning: Git committer name or email is not configured. Go to settings."
```

Comment:
Clarifying - the wording re: "settings" should be:
"Go to the Settings page to fix this."
Or maybe:
"Go to the Settings page in the Source Control menu to fix this."

The important thing is that we're pointing to "the Settings page" rather than "settings" which seems ambiguous.